openapi/paths/subscriptions@{id}@upcoming-invoices@{invoiceId}@issue.yaml

```
@@ -11,10 +11,15 @@ post:
     - Core
   tags:
     - Orders
-  summary: Issue an upcoming invoice for early pay
+  summary: Issue a selected upcoming invoice for early pay
```

suggestion: I'm not sure that 'a selected' adds to the meaning
(the description also details that the ID is required)
```suggestion
  summary: Issue an upcoming invoice for early pay
```